solidity/test/hardhat/forked-environment/ptoken-upgrade.test.js

```diff
@@ -142,12 +143,18 @@ conditionalDescribe(
           eventAttestator.sign(swapEvent),
         ]
 
-        const eventContent = swapEvent.args[1][0]
+        const decodedEvent = decodeSwapEvent(swapEvent.data)
         swapOperation = new Operation({
           blockId: swapEvent.blockHash,
           txId: swapEvent.transactionHash,
           originChainId: Chains.Bsc,
-          eventContent,
+          nonce: swapEvent.topics[1],
+          erc20: decodedEvent.erc20,
+          destinationChainId: decodedEvent.destination,
+          amount: decodedEvent.netAmount,
+          sender: decodedEvent.sender,
+          recipient: decodedEvent.recipient,
+          data: decodedEvent.data,
```

Comment:
cleaner for the reader eyes:
```suggestion
          ...decodedEvent,
          nonce: swapEvent.topics[1],
```